capa/rules/__init__.py

```diff
@@ -162,6 +203,9 @@ class Scope(str, Enum):
         capa.features.common.Class,
         capa.features.common.Namespace,
     },
+    DEV_SCOPE: {
+        capa.features.insn.API,
+    },
```

Comment:
i see this is used throughout, but i don't quite know what it means/how it should be used. i'd prefer that we only define valid scopes to be used by dynamic extractors. if thats not possible, please add a comment explaining how this should be used.

Reply:
I am working under the assumption that each rule must specify both the static and dynamic scopes, and that the features in each rule must be valid for both scopes. Because of this, I made this temporary dynamic scope that's essentially a feature sinkhole (all features are valid for this scope), with the goal of not having to alter all of our tests and make them composed of features that are only common across static and dynamic scopes. 

p.s. using the file scope wouldn't have worked since it doesn't support all the features (API for example)

Should I proceed differently with regards to these assumptions or should I add a comment explaining this scope?

Reply:
ok, thats fine, but lets plan to remove this temporary scope before the dynamic branch is merged into capa master.